packages/services/src/view-service/auctions.ts

```diff
@@ -70,8 +70,8 @@ export const auctions: Impl['auctions'] = async function* (req, ctx) {
       auction = new Any({
         typeUrl: DutchAuction.typeName,
         value: new DutchAuction({
+          state: state ?? { seq: value.seqNum },
```

Comment:
If we aren't querying latest state, we still want to return a state partial with the sequence number so that the UI can determine whether to render end/withdraw buttons.

Reply:
will this throw if state is `undefined`?

Reply:
Nope — the `??` is the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing), which defaults to the right side of the operator if `state` is null or undefined.